test/features/complexTypes.feature

```diff
@@ -1,7 +1,42 @@
 @functional @complexcollections @showandhide
-Feature: Set of scenarios testing show & hide functionality and conditional logic
```

Comment:
Looking at the scenarios here and the definition step, I think this should be the feature file for complex type override per event. And we could have a separate feature file for complex types, which won't require the  `    Given the following definition for 'complex case fields overrides'` step

Reply:
👍